spec/features/responses/form_rendering_and_submission_spec.rb

```diff
@@ -150,9 +150,11 @@
       fill_in_question([6], with: "barcode answer")
       fill_in_question([7], with: "Dog")
       fill_in_question([8], with: %w[Dog Cat])
-      fill_in_question([9], with: "Mar 12 #{Time.current.year} 18:32:44")
-      fill_in_question([10], with: "Apr 4 #{Time.current.year}")
-      fill_in_question([11], with: "03:08:23")
+      # Note: Dates and times are now selected using a javascript dialog box,
+      # so the current "fill_in_question" method does not work.
+      # fill_in_question([9], with: "#{Time.current.year}-03-12 18:32")
+      # fill_in_question([10], with: "#{Time.current.year}-04-04")
+      # fill_in_question([11], with: "03:08")
```

Comment:
general note about these commented test lines:
- like jill said, yes let's remove the comments related to time ENTRY (e.g. `fill_in_question`) since we don't need the tests for entry anymore
- but let's make sure there's at least 1 test related to time RENDERING (e.g. `expect_value`) when the time has already been saved in the database, since that's unrelated to the new picker.